### PR TITLE
[TF-1571] Apple Watch settings 

### PR DESCRIPTION
**Pull request by gmachnio**

### CODE REVIEW

File: TwoFAS/TwoFAS.xcodeproj/project.pbxproj [hunk 1]
```diff
@@ -9376,7 +9419,7 @@
 			);
 			runOnlyForDeploymentPostprocessing = 0;
 			shellPath = /bin/sh;
-			shellScript = "export PATH=\"$PATH:/opt/homebrew/bin\"\nif which swiftlint >/dev/null; then\n    swiftlint\nelse\n    echo \"warning: SwiftLint not installed, download from https://github.com/realm/SwiftLint\"\nfi\n";
+			shellScript = "#export PATH=\"$PATH:/opt/homebrew/bin\"\n#if which swiftlint >/dev/null; then\n#    swiftlint\n#else\n#    echo \"warning: SwiftLint not installed, download from https://github.com/realm/SwiftLint\"\n#fi\n";
```
Comment (adocyn) [inline TwoFAS/TwoFAS.xcodeproj/project.pbxproj:9422]: This shouldn't be commited

File: TwoFAS/TwoFAS.xcodeproj/project.pbxproj [hunk 2]
```diff
@@ -12171,7 +12217,7 @@
 				SWIFT_EMIT_LOC_STRINGS = YES;
 				SWIFT_VERSION = 5.0;
 				TARGETED_DEVICE_FAMILY = 4;
-				WATCHOS_DEPLOYMENT_TARGET = 10.4;
+				WATCHOS_DEPLOYMENT_TARGET = 10.5;
```
Comment (adocyn) [inline TwoFAS/TwoFAS.xcodeproj/project.pbxproj:12220]: Do we need to go so high?

File: TwoFAS/TwoFAS/Root/Modules/Settings/AppleWatch/View/AppleWatchView.swift [hunk 3]
```diff
@@ -0,0 +1,109 @@
...
+//  You should have received a copy of the GNU General Public License
+//  along with this program. If not, see <https://www.gnu.org/licenses/>
+//
+
+import SwiftUI
+import Common
+
+struct AppleWatchInstallationStep: Hashable, Identifiable {
+    var id = UUID()
+    let descirption: String
```
Comment (adocyn) [inline TwoFAS/TwoFAS/Root/Modules/Settings/AppleWatch/View/AppleWatchView.swift:25]: `description` -> typo

File: TwoFAS/TwoFAS/Root/Modules/Settings/AppleWatch/View/AppleWatchViewController.swift [hunk 4]
```diff
@@ -0,0 +1,49 @@
...
+            self?.presenter.handleInstallationStep(number: stepNumber)
+        }
+        let appleWatchView = AppleWatchView(
+            appleWatchInstallationSteps: [
+                .init(descirption: T.AppleWatch.installationFirstStep,
+                      actionTitle: T.AppleWatch.installationFirstStepLink),
+                .init(descirption: T.AppleWatch.installationSecondStep,
+                      actionTitle: T.AppleWatch.installationSecondStepLink)
+            ],
+            installationStepActionCallback: installationStepAction
```
Comment (adocyn) [inline TwoFAS/TwoFAS/Root/Modules/Settings/AppleWatch/View/AppleWatchViewController.swift:39]: This isn't necessary. We should pass AppleWatchPresenter directly into AppleWatchView and call `onInstallationStep(....` from there.

File: TwoFAS/TwoFAS/Root/Modules/Settings/AppleWatch/View/AppleWatchView.swift [hunk 5]
```diff
@@ -0,0 +1,109 @@
...
+                .renderingMode(.original)
+                .padding(.top, Theme.Metrics.standardSpacing)
+
+            Text(T.AppleWatch.installationInfoTitle)
+                .font(Font(Theme.Fonts.Text.title))
+                    .multilineTextAlignment(.center)
+                    .minimumScaleFactor(0.7)
+
+            VStack(alignment: .leading) {
+                ForEach(Array(appleWatchInstallationSteps.enumerated()), id: \.element) { index, step in
```
Comment (adocyn) [inline TwoFAS/TwoFAS/Root/Modules/Settings/AppleWatch/View/AppleWatchView.swift:47]: I would check that on smaller screens as a Scroll View can be necessary here

**Review by adocyn — CHANGES_REQUESTED** [review]

File: TwoFAS/TwoFAS.xcodeproj/project.pbxproj (hunk 2 above)
Comment (gmachnio) [reply TwoFAS/TwoFAS.xcodeproj/project.pbxproj:12220]: No, old change. I'll revert it.

File: TwoFAS/TwoFAS/Root/Modules/Settings/AppleWatch/View/AppleWatchView.swift (hunk 5 above)
Comment (gmachnio) [reply TwoFAS/TwoFAS/Root/Modules/Settings/AppleWatch/View/AppleWatchView.swift:47]: Screenshot 2024-07-22 at 23 01 41

Comment (gmachnio) [reply TwoFAS/TwoFAS/Root/Modules/Settings/AppleWatch/View/AppleWatchView.swift:47]: IMO looks good on SE 3rd generation.

**Review by adocyn — APPROVED** [review]

**Pull request merged**